configs/app/meta.ts

```diff
@@ -0,0 +1,12 @@
+import app from './app';
+import { getEnvValue } from './utils';
+
+const defaultImageUrl = app.baseUrl + '/static/og_placeholder.png';
+
+const meta = Object.freeze({
+  promoteBlockscout: getEnvValue(process.env.NEXT_PUBLIC_PROMOTE_BLOCKSCOUT_IN_TITLE) || 'true',
```

Comment:
Let's be more specific here. which title we are talking about - page, tab, <title>, `og:title`, something else?

Reply:
I think that since it's "meta" config, it's obvious that we're talking about meta titles. all of them, title tag and og title tag.